packages/terra-list/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,127 @@
+# Terra List - Sections with Subsections
+
+With the inclusion of sections in the list, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. The guidance from UX is to have both sections and subsections to be collapsible.
```

Comment:
Should the last sentence be, The guidance from UX is to **not** have both sections and subsections to be collapsible, _have one or the other be collapsible_.

Reply:
Ha, wow, that was a big typo!

Reply:
Updated.